src/_pytest/python.py

```diff
@@ -288,7 +289,7 @@ def getmodpath(self, stopatmodule=True, includemodule=False):
         s = ".".join(parts)
         return s.replace(".[", "[")
 
-    def reportinfo(self):
+    def reportinfo(self) -> Tuple[str, int, str]:
```

Comment:
Not sure if this `reportinfo` and `Item`'s `reportinfo` are related, but their types are slightly different (`int` vs. `Optional[int]`).

Reply:
They are related.. the base allows for None, but here it is always int.
Should be ok, no?

Reply:
Yes, a subclass can be more specific in a return type.